tests/functional/ubuntu/16.04/mon/initial_members/vagrant_variables.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+# DEPLOY CONTAINERIZED DAEMONS
+docker: false
+
+# DEFINE THE NUMBER OF VMS TO RUN
+mon_vms: 1
```

Comment:
Yes I know but the `vagrant provision` won't work. But I just realized we were running ansible manually so this won't be an issue :)